scapy/layers/tls/session.py

```diff
@@ -85,6 +86,12 @@ def __init__(self,
             ciphersuite = TLS_NULL_WITH_NULL_NULL
         self.ciphersuite = ciphersuite(tls_version=tls_version)
 
+        if not self.ciphersuite.usable: # cryptography isn't installed ?
```

Comment:
I think that it is better to display an error message, hence the patch could be:
```
if not self.ciphersuite.usable:
                warning("TLS ciphersuite not useable. Is the cryptography Python module installed ?")
                 return
```